styles/main.css

```diff
@@ -132,8 +148,16 @@ footer hr{
 }
 
 .back-to-top{
-	    margin-right: -.5em;
+	margin-right: -.5em;
+    text-align: right;
     opacity: .5;
-	float:right;
+}
+
+.back-to-top a {
+    position: fixed;
+    bottom: 3.5em;
 }
 	
+.back-to-top:hover {
+    opacity: 1;
+}
```

Comment:
Missing rule to make increase the opacity when the back-to-top link is being hovered